File: abc387/b/b.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(){
  int n;
  cin >> n;
  int ans = 0;
  for(int i = 1; i <= 9; i++)for(int j = 1; j <= 9; j++)if(i*j != n) ans += i*j;
  cout << ans << endl;
}

File: abc387/b/b.go
package main

import "fmt"

func main() {
	var x, ans int
	fmt.Scan(&x)
	for i := 1; i < 10; i++ {
		for j := 1; j < 10; j++ {
			if i*j != x {
				ans += i * j
			}
		}
	}
	fmt.Println(ans)
}
